template-quality-checklist.md

```diff
@@ -0,0 +1,102 @@
+# Template quality checklist
+
+This quality checklist will eventually become a pull request template that every contributor needs to add to pull requests against the `templates` repository.
+
+
+# Pull request summary
+
+{Describe the purpose of your pull request.}
+
+## Template contributor checklist
+
+IMPORTANT: The next three sections in this checklist should be filled out by the template contributor at the time they submit this request.
+
+NOTE: Pull requests can only be merged when all boxes are checked.
+
+### Which issue does this pull request fix or reference?
+
+This pull request:
+
+- Resolves: {Choose this option if this PR will close the issue and link to issue}
+- Relates to: {Choose this option if this PR only references an issue for context but does not fully resolve it}
+
+### Procedural requirements
+
+- [ ] The template contributor participated in a template working group to create this template.
+- [ ] This template was workshopped with the community.
+- [ ] A template mentor officially approved this template set to move to the pull request phase.
+
+
+### Template set requirements
+
+- [ ] Raw template is present.
+- [ ] Template guide is present.
+- [ ] Template before-you-start guide is present (optional).
+- [ ] Template example is present OR a new issue was created to track this task (include link).
+
+
+## Template pull request reviewer checklist
+
+IMPORTANT: The rest of the sections in this checklist should only be filled out by authorized Good Docs Project pull request reviewers. If you are the individual template contributor, do not fill out the rest of the fields or check the boxes.
+
+NOTE: Pull requests can only be merged when all boxes are checked.
+
+
+### Mechanics and formatting requirements - PULL REQUEST REVIEWER ONLY
+
+- [ ] Check rendered Markdown output to ensure it renders correctly.
+- [ ] Review the template set to ensure all documents follow the template markdown style guide.
+- [ ] The template set is free from grammar errors and typos.
+
+
+### Overall usability - PULL REQUEST REVIEWER ONLY
+
+- [ ] The scope of the template set is appropriate, meaning it is not too simple or overly complex (i.e. it needs to be expanded or broken into multiple smaller templates).
+- [ ] The template is complete and comphrehensive.
+- [ ] The template set is well-organized and the contents flow in a logical order.
+- [ ] The template is well-written and clear.
+- [ ] The template set provides sufficient guidance about how to fill it in and implement the template in a documentation project.
+
+
+### Raw template requirements - PULL REQUEST REVIEWER ONLY
+
+- [ ] Check rendered Markdown output to ensure it renders correctly and follows the template markdown style guide.
+- [ ] The template title is present and is an H1.
+- [ ] The template includes an introductory comment pointing them to other template resources.

```

Comment:
Is "pointing to other template resources" a requirement for the introductory comment? I did not think so, from the description provided earlier.

Reply:
My thinking here is that if they only land on the template file itself and don't realize there are supporting documents (like a guide), they need to know those supporting docs exist. I could make sure that's included in the template instructions.

Reply:
I've added some clarity to this section per your feedback. Resolving.